stringfuzz/transformers/translate.py

```diff
@@ -28,15 +29,17 @@ def make_table(self, character_set):
         character_set = ''.join(character_set)
         return str.maketrans(character_set, shuffled)
 
-    def exit_literal(self, literal):
+    def exit_literal(self, literal, parent):
         if isinstance(literal, StringLitNode):
+            if isinstance(parent, ReRangeNode) and not self.re_range_flag:
+                return
             literal.value = literal.value.translate(self.table)
 
 # public API
-def translate(ast, integer_flag):
+def translate(ast, integer_flag, re_range_flag):
     if integer_flag:
         character_set = WITH_INTEGERS
     else:
         character_set = WITHOUT_INTEGERS
-    transformed = TranslateTransformer(ast, character_set).walk()
+    transformed = TranslateTransformer(ast, character_set, re_range_flag).walk()
```

Comment:
Nitpick: naming, same as for `multiply.py`.

Reply:
That's not ok?

Reply:
It's ok this way too. I thought `skip_re_range` might be more descriptive, but this is also fine. Feel free to merge it either way.